src/components/game/game.jsx

```diff
@@ -1,60 +1,23 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Container, Button } from 'react-bootstrap';
 
 import Board from '../board';
+import calculateWinner from '../../utilities';
 
-class Game extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      history: [
-        {
-          squares: Array(9).fill(null),
-        },
-      ],
-      stepNumber: 0,
-      xIsNext: true,
-    };
-  }
-
-  handleClick(int) {
-    const history = this.state.history.slice(0, this.state.stepNumber + 1);
-     const current = history[history.length - 1];
-     const squares = current.squares.slice();
-    if (calculateWinner(squares) || squares[int]) {
-      return;
-    }
-    squares[int] = this.state.xIsNext ? 'X' : 'O';
-    this.setState({
-      history: history.concat([
-        {
-          squares,
-        },
-      ]),
-      stepNumber: history.length,
-      xIsNext: !this.state.xIsNext,
-    });
-  }
-
-  jumpTo(step) {
-    this.setState({
-      stepNumber: step,
-      xIsNext: step % 2 === 0,
-    });
-  }
-  render() {
-    const { history } = this.state;
-    const current = history[this.state.stepNumber];
+const Game = (props) => {
```

Comment:
Should be destructioring the props here e.g. 
``const Game = ({ 
history,
stepNumber,
jumpTo,
etc...
}) => {